packages/visx-brush/src/Brush.tsx

```diff
@@ -60,6 +61,8 @@ export type BrushProps = {
   innerRef?: React.MutableRefObject<BaseBrush | null>;
   /** Prevent drag end on mouse leaving from brush stage. */
   useWindowMoveEvents?: boolean;
+  /** Render function for custom brush handles */
```

Comment:
Nit – just adding a period since docs are generated from these.

```suggestion
  /** Render function for custom brush handles. */
```